migrations/2022-11-27-180447_custom_emojis/up.sql

```diff
@@ -0,0 +1,16 @@
+create table custom_emoji (
+  id serial primary key,
+  local_site_id int references local_site on update cascade on delete cascade not null,
+  shortcode varchar(128) not null UNIQUE,
+  image_url text not null UNIQUE,
+  alt_text text not null,
+  category text not null,
+  published timestamp without time zone default now() not null,
+  updated timestamp without time zone
+);
+
+create table custom_emoji_keyword (
+  id serial primary key,
+  custom_emoji_id int references custom_emoji on update cascade on delete cascade not null,
+  keyword varchar(128) not null
+);
```

Comment:
Forgot another thing, this is going to allow you to insert a ton of the same keyword for each emoji. You need to add a unique constrait on `(custom_emoji_id, keyword)`

Reply:
Added. Will add validation on frontend later.